src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,64 @@
+package duke;
+import duke.command.Command;
+import javafx.application.Platform;
+
+
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private UiManager uiManager;
+    public Duke() {
+        this("data/tasks.txt");
+    }
+
+    /**
+     * Constructor of Duke class.
```

Comment:
Iirc the default style of javadoc recommends to add an empty line between the description and the parameter section 🤔